docs/SimpleSelect.md

```diff
@@ -0,0 +1,9 @@
+# SimpleSelect
+
+## Props
+| Name            | Type     | Default     | Required | Description                                                         |
+| --------------- | -------- | ----------- | -------- | ------------------------------------------------------------------- |
+| onChange        | Function | () => false | false    |                                                                     |
+| value           | any      | null        | false    | value passed into `value`  prop of `SimpleSelect`                   |
```

Comment:
this seems to be badly defined and can be error prone - 'any' is also a bad type, we must be more specific